Jenkinsfile

```diff
@@ -104,7 +114,8 @@ if( buildParams.BUILD_OS != "all" ){
 
 // Using .each{} hits "a CPS-transformed closure is not yet supported (JENKINS-26481)"
 for( label in build_nodes) {
-  stage_unit_test(label)
-  stage_rpmbuild(label)
-  stage_integration(label)
+//stage_unit_test(label)
+//stage_rpmbuild(label)
+//stage_integration(label)
+  stage_clean_rpms(label)
 }
```

Comment:
I agree with @unakatsuo on this one.

It's true that the main Jenkinsfile runs tests generally unrelated to this branch but if it gets merged with those lines commented out, it will stop running the tests on master (and future branches) too. It's ok to comment these out during development but before merge, it should be restored back to how it was.

Also in general I think it's good to run tests at least once before merging even if unrelated because you just never know. Some times unrelated code can have unexpected consequences.